lib/recurly/coupon.php

```diff
@@ -56,6 +68,17 @@ public static function deleteCoupon($couponCode, $client = null) {
     return Recurly_Base::_delete(Recurly_Coupon::uriForCoupon($couponCode), $client);
   }
 
+  protected function createUpdateXML() {
```

Comment:
i think this should be a public method and should have a unit test on it... like we do with `xml()`